packages/eas-cli/src/build/android/version.ts

```diff
@@ -39,3 +118,39 @@ export async function maybeResolveVersionsAsync(
     };
   }
 }
+
+async function writeVersionsToBuildGradleAsync({
+  projectDir,
+  exp,
+}: {
+  projectDir: string;
+  exp: ExpoConfig;
+}): Promise<string> {
+  const buildGradle = readBuildGradle(projectDir);
+  if (!buildGradle) {
+    throw new Error('missing build gradle');
```

Comment:
The error message is user-facing so please make it more friendly. Use full sentences. 
Also `build gradle` -> `build.gradle`